usefulUtilities/statusIndicator/README.md

```diff
@@ -0,0 +1,71 @@
+# Status Indicator
+Let other avatars know your status!
+
+Only available in desktop mode, creates a clickable status in upper right hand corner of screen to set status to available/busy. 
+
+A web app and database tracks these statuses. Using a web entity, you can display everyone's status in a table by going to:
+
+To see everyone's status:
+`https://<hosted web app url>/getAllEmployees`
+
+To see a team's list status:
+`https://<hosted web app url>/?type=getTeamEmployees&teamName=ExampleTeamName`
+
+
+## Setup
+
+Run `<statusIndicator root>/hifiScript/statusIndicator_webApp.js` in High Fidelity's Running scripts.
```

Comment:
Is this supposed to refer to the web app?

Reply:
nope!